integration/log.go

```diff
@@ -24,6 +24,7 @@ import (
 	"strings"
 	"time"
 
+	"github.com/cenkalti/backoff"
```

Comment:
This package has an MIT license, so that's not an issue. I could use `client/backoff`, and I don't think its location is problematic since the integration test is acting as a client. I'd have to write more code though, since our backoff package is extremely minimal compared to this one. This lets me do things like "retry this function with exponential backoff until the context deadline is reached", which is rather nice. Really, I'd prefer to delete our package in favour of this one.